packages/webpack/src/loader.ts

```diff
@@ -49,18 +42,34 @@ function createIdentifier(
   return `./${fileIdent}-${lang}-virtual.json!=!${unloader}!`;
 }
 
-const renderLanguageLoaderAsync =
-  (resourcePath: string, loadedTranslation: LoadedTranslation) =>
-  (lang: string) => {
-    const identifier = createIdentifier(lang, resourcePath, loadedTranslation);
-
-    return `'${encodeWithinSingleQuotes(
-      lang,
-    )}': createLanguage(require.resolveWeak('${identifier}'), () => import(
-      /* webpackChunkName: "${getChunkName(lang)}" */
-      '${identifier}'
-    ))`;
+// reimplement `stringifyRequest` from loader-utils 2.x
+// https://github.com/webpack/loader-utils/blob/master/CHANGELOG.md#300-2021-10-20
+function stringifyRequest(this: LoaderContext, request: string) {
+  return JSON.stringify(this.utils.contextify(this.context, request));
+}
+
+function renderLanguageLoaderAsync(
+  this: LoaderContext,
+  resourcePath: string,
+  loadedTranslation: LoadedTranslation,
+) {
+  return (lang: string) => {
+    const identifier = stringifyRequest.call(
+      this,
+      createIdentifier(lang, resourcePath, loadedTranslation),
+    );
+
+    return /* ts */ `
```

Comment:
What is this business?

<img width="523" alt="ts-comment-1" src="https://user-images.githubusercontent.com/3297808/228144399-edbe9f4b-5490-4289-a44a-989c5b758dd8.png">

<img width="523" alt="ts-comment-2" src="https://user-images.githubusercontent.com/3297808/228144414-081cd394-1d7b-4b50-96c6-8fb98694afcb.png">

Reply:
Feel free to remove that gitignore